### ISSUE TITLE
Can not see task that we create in calendar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#659]
```
What steps will reproduce the problem?
1.Open the task pane in the calendar view
2.Create a task 
3.I received an email confirmation of the task with a date of 1969 and I can 
not view the task anywhere. 

What is the expected output? What do you see instead?
All tasks should be displayed in the list of task or offer option to show all

What version of the product are you using? On what operating system?
Linux latest


Please provide any additional information below.


```

Original issue reported on code.google.com by `bgauthie...@gmail.com` on 10 Apr 2014 at 12:45

**Comment by GoogleCodeExporter** [comment]
```
Well, I won't fix it. Tasks of 1969 ????
```

Original comment by `myroundc...@gmail.com` on 16 Apr 2014 at 2:58
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Fixed (Roundcube 1.0 branch only)
```

Original comment by `myroundc...@gmail.com` on 21 Apr 2014 at 6:47
